tests/test_mortgage_rate_crud.py

```diff
@@ -0,0 +1,125 @@
+import pry
+import json
+from tests.BaseCase import BaseCase
+from database.mortgage_rate import range_620_639, range_640_659, range_660_679, range_680_699, range_700_759, range_760_850
```

Comment:
Great job testing! Love the import of data from a separate file - helps with readability!